cli/main.rs

```diff
@@ -655,11 +664,95 @@ async fn test_command(
   global_state
     .file_fetcher
     .save_source_file_in_cache(&main_module, source_file);
+
+  let mut maybe_coverage_collector = if coverage {
+    let inspector = match worker.inspector.as_mut() {
+      Some(inspector) => inspector,
+      None => {
+        return Err(
+          // TODO(caspervonb) this should be implicit when the cover flag is provided.
```

Comment:
More specifically, if there has been no request for an inspector server we should create an inspector that doesn't have a websocket server attached to it.

That requires quite a bit of refactoring of inspector.rs, leaving for a future date.

Reply:
This check is not required since you got `requires("inspect")` in the arg configuration.